scripts/app.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+var projects = [];
+
+function Projects (options) {
```

Comment:
Good practice to keep constructors names as singular. Everything else looks great!